### PR TITLE
Several BZ fixes for 3.9

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses:
https://bugzilla.redhat.com/show_bug.cgi?id=1557404
https://bugzilla.redhat.com/show_bug.cgi?id=1555436
https://bugzilla.redhat.com/show_bug.cgi?id=1556836
https://bugzilla.redhat.com/show_bug.cgi?id=1557960
https://bugzilla.redhat.com/show_bug.cgi?id=1558488
https://bugzilla.redhat.com/show_bug.cgi?id=1558430

@sdodson Please check behind me for accuracy. Thanks!

### CODE REVIEW

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -170,18 +170,28 @@ with {product-title} 3.9:
...
 +
-This change is mainly so that the web console, which used to run as part of the
-master itself, can instead be run as a pod deployed to the master.
+----
+# cat /etc/origin/master/master-config.yaml|grep "defaultNodeSelector"
+  defaultNodeSelector: node-role.kubernetes.io/compute=true
+----
+
+.. Add the `node-role.kubernetes.io/compute=true` label for non-infra and
+non-masters nodes.
```
Comment (ganhuang) [inline upgrading/automated_upgrades.adoc]: @ahardin-rh Hi, Ashley. 

The steps above should be part of manual upgrade, NOT automated upgrade. Also the command and statement is a draft only as it's really complicated and challenging that QE can't offer the details.

File: install_config/install/disconnected_install.adoc [hunk 2]
```diff
@@ -345,6 +345,8 @@ $ docker save -o ose3-images.tar \
     registry.access.redhat.com/openshift3/ose-keepalived-ipfailover \
     registry.access.redhat.com/openshift3/ose-pod \
     registry.access.redhat.com/openshift3/ose-sti-builder \
+    registry.access.redhat.com/openshift3/ose-template-service-broker \
+    registry.access.redhat.com/openshift3/ose-web-console
```
Comment (sdodson) [inline install_config/install/disconnected_install.adoc]: Need to add `registry.access.redhat.com/openshift3/ose-ansible-service-broker`

**Review by sdodson — CHANGES_REQUESTED** [review]
Just that one missing image otherwise looks good.

File: upgrading/automated_upgrades.adoc (hunk 1 above)
Comment (sdodson) [reply upgrading/automated_upgrades.adoc]: Yes, @ganhuang is correct, this is handled automatically via the playbooks.

**Comment by ahardin-rh** [comment]
Thanks @sdodson Fixed :hammer_and_wrench:

**Review by sdodson — APPROVED** [review]
looks good with recent changes, thanks

**Pull request merged**